client/src/views/my-program-items/components/MyDirectSignupsList.tsx

```diff
@@ -35,10 +38,10 @@ export const MyDirectSignupsList = ({
   }, [missedSignups, directSignups]);
 
   return (
-    <RaisedCard data-testid="direct-signup-program-items-list">
```

Comment:
Oliko tää `data-testid` pudonnut pois? Playwright-testit käyttää näitä.

Reply:
Joo, jossain refaktoroinnin hulinoissa se oli hukkunut, nyt fiksattu :)